Sources/ViewModels/ALKConversationViewModel.swift

```diff
@@ -1215,6 +1232,37 @@ open class ALKConversationViewModel: NSObject, Localizable {
             }
         })
     }
+    
+    func loopOverTheLoadedMessageArray() {
+        if count >= alMessages.count {
+            return
+        }
```

Comment:
It's not clear what the function is doing.

Reply:
this method is to show typing indicator for Welcome message.

Reply:
Then change its name to `showTypingIndicatorForWelcomeMessage` or something. Or make it generic and use it for both welcome message and normal message.

Reply:
updated